src/components/Tab/Tab.js

```diff
@@ -33,47 +42,42 @@ const StyledTab = styled.div`
     content: '';
     position: absolute;
     width: calc(100% - 4px);
-    height: 4px;
+    height: 6px;
 
     background: ${({ theme }) => theme.material};
-    bottom: -2px;
+    bottom: -3px;
     left: 2px;
   }
 `;
-const Tab = ({
-  value,
-  onClick,
-  active,
-  children,
-  className,
-  style,
-  ...otherProps
-}) => (
-  <StyledTab
-    className={className}
-    active={active}
-    style={style}
-    {...otherProps}
-    onClick={() => onClick(value)}
-  >
-    {children}
-  </StyledTab>
-);
+
+// TODO handle tabIndex
+const Tab = React.forwardRef(function Tab(props, ref) {
```

Comment:
Why are you using a named function called `Tab`? (If you still want to keep a named function maybe it could be renamed so it won't conflict with the `Tab` variable declared in the outer scope)

Reply:
This doesn't break anything. I am using similar approach as seen in [Material UI components](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Card/Card.js). In React devtools the component name is the name of the function passed to `forwardRef`. On the other hand the propTypes are declared on `const` so renaming it to some other arbitrary name would be kind of confusing

Reply:
ok cool